tests/HtaccessTest.php

```
@@ -0,0 +1,402 @@
+
+        $precedingContent = substr($this->content, 0, $uploadsRule);
+        $nearestIfModuleOpen = strrpos($precedingContent, '<IfModule mod_rewrite.c>');
+        $this->assertNotFalse(
+            $nearestIfModuleOpen,
+            'The uploads RewriteRule must be preceded by an <IfModule mod_rewrite.c> guard.'
+        );
+
+        $ifModuleClose = strpos($this->content, '</IfModule>', $uploadsRule);
+        $this->assertNotFalse($ifModuleClose, 'The uploads RewriteRule guard must be closed with </IfModule>.');
+
+        $nearestClosePriorToRule = strpos($precedingContent, '</IfModule>', $nearestIfModuleOpen);
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $nearestClosePriorToRule. Keep variable name length under 20. (undefined)

(LongVariable)